build.xml

```diff
@@ -27,6 +27,11 @@
         <!-- Packages required by mandatory JWNL -->
         <pathelement location="${PREFIX}/commons-logging.jar"/>
 
+        <!-- Logging -->
+        <pathelement location="${PREFIX}/slf4j-api-1.7.25.jar"/>
```

Comment:
Don't think specific version of .jar should be used, I would use soft link instead, for instance:
```
<pathelement location="${PREFIX}/slf4j-api.jar"/>
```
Because version specific link will be broken when package version is updated.

Reply:
I have removed versions from the sh/bat scripts as well.